src/tools/PrizeTierController/hooks/useAllPrizeTierHistoryData.ts

```diff
@@ -19,18 +20,33 @@ export const useAllPrizeTierHistoryData = () => {
   if (isFetched) {
     prizeTierHistoryContracts.forEach((contract, i) => {
       const prizeTier = results[i].data
-      if (allPrizeTierHistoryData[contract.chainId] === undefined) {
-        allPrizeTierHistoryData[contract.chainId] = {}
-      }
-      allPrizeTierHistoryData[contract.chainId][contract.address] = {
-        bitRangeSize: prizeTier.bitRangeSize,
-        expiryDuration: prizeTier.expiryDuration,
-        maxPicksPerUser: prizeTier.maxPicksPerUser,
-        prize: prizeTier.prize,
-        tiers: prizeTier.tiers,
-        endTimestampOffset: prizeTier.endTimestampOffset
+      if (!!prizeTier) {
+        if (allPrizeTierHistoryData[contract.chainId] === undefined) {
+          allPrizeTierHistoryData[contract.chainId] = {}
+        }
+        if (contract.isV2) {
+          allPrizeTierHistoryData[contract.chainId][contract.address] = {
```

Comment:
The `PrizeTierV2` type also includes `drawId`, which isn't in `PrizeTierConfigV2`. This _**probably**_ wouldn't cause any issues, except for maybe when `drawId` is appended to it from the users' input in `PrizePoolTransactionDisplay`.

In general, I just wanted to avoid any future bugs that could be caused by carrying over an extra value like that.